src/ripple/app/main/Application.cpp

```diff
@@ -824,24 +824,145 @@ class ApplicationImp
     beast::Journal journal (std::string const& name) override;
 
     //--------------------------------------------------------------------------
-    bool initSqliteDbs ()
+
+    bool
+    initDBs()
```

Comment:
Conceptually, I see SQLite and nodestore databases as distinct. My preference is to have separate functions called within Application:setup() that each pertain to initializing/tuning these things accordingly. I agree that it's a good idea to move these activities from the body of setup(), and I think that it would make it even easier to reason about if there were 2 functions, such as initSqliteDbs() for SQL, and initNodestoreDbs() for nodestore stuff.